README.md

```diff
@@ -1,16 +1,31 @@
 ![Lifecycle:Experimental](https://img.shields.io/badge/Lifecycle-Experimental-339999)
 
-# GDX Agreements Tracker Front End
+# GDX Agreements Tracker
 
 ## Installation
```

Comment:
Would it make more sense to have Front end and api as h2 and make the install and build as h3 within them